extensions/gulp/src/main.ts

```diff
@@ -122,12 +122,12 @@ class FolderDetector {
 		if (platform === 'win32' && await exists(path.join(rootPath!, 'node_modules', '.bin', 'gulp.cmd'))) {
 			const globalGulp = path.join(process.env.APPDATA ? process.env.APPDATA : '', 'npm', 'gulp.cmd');
```

Comment:
I think we only need the quotes here, since the other paths are guaranteed to not have spaces.

Reply:
agree, will provide an updated PR

Reply:
I tried it out, and the `exists` check fails now. Looks like the quotes need to be moved to line 125:
`gulpCommand = '"' + globalGulp + '"';`